test/oli_web/live/curriculum/container_test.exs

```diff
@@ -42,10 +44,60 @@ defmodule OliWeb.Curriculum.ContainerLiveTest do
       assert view |> element("##{Integer.to_string(page1.id)}") |> has_element?()
       assert view |> element("##{Integer.to_string(page2.id)}") |> has_element?()
     end
+
+    test "shows duplicate action for pages", %{
+      conn: conn,
+      author: author,
+      project: project,
+      map: map
+    } do
+      conn =
+        recycle(conn)
+        |> Pow.Plug.assign_current_user(author, OliWeb.Pow.PowHelpers.get_pow_config(:author))
+        |> get("/authoring/project/#{project.slug}/curriculum/")
+
+      {:ok, view, _html} = live(conn)
+
+      revision_page_one = Map.get(map, :revision1)
+      revision_page_two = Map.get(map, :revision2)
+
+      # Duplicate action is present with the right revision id
+      assert view
+        |> element("div[phx-value-slug=\"#{revision_page_one.slug}\"] button[phx-click=\"duplicate_page\"]")
+        |> render =~ "phx-value-id=\"#{revision_page_one.id}\""
+
+      # Clicking on duplicate action creates a new entry with the right title name
+      view
+        |> element("div[phx-value-slug=\"#{revision_page_two.slug}\"] button[phx-click=\"duplicate_page\"]")
+        |> render_click =~ "entry-title\">Copy of #{revision_page_two.title}</span>"
+    end
+
+    test "does not show duplicate action for adaptive pages", %{
+      conn: conn,
+      author: author,
+      project: project,
+      map: map
+    } do
+      conn =
+        recycle(conn)
+        |> Pow.Plug.assign_current_user(author, OliWeb.Pow.PowHelpers.get_pow_config(:author))
+        |> get("/authoring/project/#{project.slug}/curriculum/")
+
+      {:ok, view, _html} = live(conn)
+
+      adaptive_page_revision = Map.get(map, :adaptive_page_revision)
+
+      assert view
```

Comment:
NIT: you can just use [has_element?/3](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveViewTest.html#has_element?/3) here